chrome/skin/style.css

```diff
@@ -41,8 +41,21 @@ textbox#urlbar {
 #universal-search-recommendation-image,
 #universal-search-recommendation-letterbox {
   border-radius: 3px;
+  box-shadow: inset 0 0 0 1px rgba(0, 0, 0, 0.1);
   height: 32px;
   margin-right: 8px;
+  overflow: hidden;
+  width: 32px;
+}
+
+#universal-search-recommendation-logo,
+#universal-search-recommendation-keyimage {
+  position: relative;
+  z-index: -1;
```

Comment:
This is weird. In order for the inset shadow of the wrapper to overlay the image contained by its children, it needs to have a negative z-index. In order for z-index to apply, it needs to be relatively positioned.

Example of what it allows:

<img width="195" alt="screen shot 2016-03-08 at 1 58 54 pm" src="https://cloud.githubusercontent.com/assets/23885/13616189/07ba4b2c-e536-11e5-8e83-6c0d9904580b.png">